overview/typescript.md

```diff
@@ -0,0 +1,38 @@
+---
+title: TypeScript
+description: TypeScript is an open-source language which builds on JavaScript
+sidebar_position: 8
+---
+
+## Overview
+
+TypeScript is an open-source language which builds on JavaScript, one of the world’s most used tools, by adding static type definitions.
+
+Types provide a way to describe the shape of an object, providing better documentation, and allowing TypeScript to validate that your code is working correctly.
+
+## TypeScript and Angular
+
+Angular applications can only be built with TypeScript.
+As Angular is the predominant language used for PlaceOS Frontend Applications this makes TypeScript a requirement.
+
+### Versioning
+
+For TypeScript, the version is dictated by the Angular version.
+
+Angular 12 (currently used by PlaceOS Frontends) supports TypeScript version 4.2.4.
+
+## TypeScript References and Tooling
+
+The [TypeScript Handbook](https://www.TypeScriptlang.org/docs/handbook) provides detailed documentation specifically related to TypeScript. 
+
+PlaceOS use [NX](https://nx.dev/) which is a suite of powerful tools to assist with testing and building JavaScript Applications.
+
+NX adds [Jest](https://jestjs.io/) for Unit Testing and [Cypress](https://www.cypress.io/) for Integration and End-to-End Testing.
+
+The [Angular CLI](https://angular.io/cli) is a command-line interface that lets you develop, scaffold and maintain Angular applications and is used with PlaceOS Frontend Applications.
```

Comment:
Is this referring to https://github.com/PlaceOS/frontends ? If so it should be pluralized in both instances where it's marked as an error

Reply:
No, not specifically, its just referring to the tool and the fact that we leverage it.

Reply:
sorry, submitted a comment separate to review. In the review comment I made it clearer I'm referring to "Fronted Applications" - if it is applications in general then it should be lower case, if it is PlaceOS Frondends, it should be pluralised